presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -58,6 +60,251 @@ General Properties
+    * **Default value:** ``all-at-once``
+
+    Setting this value to ``phased`` will allow the query scheduler to split a single
+    query execution between different time slots. This will allow Presto to switch context
+    more often and possibly stage the partially executed query in order to increase robustness.
+    Average time to execute a query may slightly increase after setting this to ``phased``,
+    but query execution time will be more consistent. This can also be specified on a
+    per-query basis using the ``execution_policy`` session property.
+
+
+``query.initial-hash-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Minimum Value:** ``1``
+    * **Default value:** ``100``
+
+    This value is used to determine how many nodes may share the same query when fixed
+    partitioning is chosen by Presto. Manipulating this value will affect the distribution
+    of work between nodes. A value lower then the number of Presto nodes may lower the utilization
+    of the cluster in a low traffic environment. An excessively high value will cause multiple
+    partitions of the same query to be assigned to a single node, or Presto may ignore
+    the setting if ``node-scheduler.multiple-tasks-per-node-enabled`` is set to false -
+    the value is internally capped at the number of available worker nodes in such scenario.
+    This can also be specified on a per-query basis using the ``hash_partition_count``
+    session property.
+
+
+``query.low-memory-killer.delay``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Minimum Value:** ``5s``
+    * **Default value:** ``5m``
+
+    Delay between a cluster running low on memory and invoking a query killer.
+    A lower value may cause more queries to fail fast, but fewer queries to
+    fail in an unexpected way.
+
+
+``query.low-memory-killer.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``false``
+
+    This property controls whether a query killer should be triggered when a cluster
+    is running out of memory. The killer will drop the largest queries first so enabling
+    this option may cause problems with executing large queries in a highly loaded cluster,
+    but should increase stability of smaller queries.
+
+
+``query.manager-executor-pool-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Minimum Value:** ``1``
+    * **Default value:** ``5``
+
+    Size of the thread pool used for garbage collecting after queries. Threads from this
+    pool are used to free resources from canceled queries, as well as enforce memory limits,
+    queries timeouts etc. More threads will allow for more efficient memory management,
+    and so may help avoid out of memory exceptions in some scenarios. However, having more
+    threads may also increase CPU usage for garbage collecting and will have an additional
```

Comment:
`CPU usage for garbage collecting` -> this is confusing and made me think about the JVM garbage collection.

I feel like we shouldn't use the phrase "garbage collection" here as it may be confusing for some people.